internal/oci/runtime_pod.go

```diff
@@ -171,6 +176,7 @@ func (r *runtimePod) ExecSyncContainer(ctx context.Context, c *Container, cmd []
 		Command:  cmd,
 		Timeout:  uint64(timeout),
 		Terminal: c.terminal,
+		Stdin:    c.stdin,
```

Comment:
one nit: I don't think we need this, I opened https://github.com/containers/conmon-rs/pull/750